e2e/union_find_test/test_cartesian/test.yml

```
@@ -0,0 +1,53 @@
+query: >
+  MATCH (m {id: 0}), (n {id: 2}), (o {id: 1}), (p {id: 3})
+  CALL union_find.connected([m, o], [n, p], "c") YIELD *
+  RETURN node1, node2, connected;
```

```suggestion
query: >
  MATCH (m {id: 0}), (n {id: 2}), (o {id: 1}), (p {id: 3})
  CALL union_find.connected([m, o], [n, p], "c") YIELD *
  RETURN node1.id as node1_id, node2.id as node2_id, connected
  ORDER BY node1_id, node2_id;
```

I suggest these changes because of output simplicity, as well as deterministic order

---

Nice catch, done!